### PR TITLE
task VAA-3_5 complete

### PR DESCRIPTION
**Pull request by hnamazli**

### CODE REVIEW

**Review by Musa-94 — COMMENTED** [review]
`classes: ['simulator__input']` you can’t just: `class: 'simulator__input'` ???

**Comment by hnamazli** [comment]
unfortunately not my friend because we have classes come through the loop

File: src/view.js [hunk 1]
```diff
@@ -15,6 +15,24 @@ class View {
...
+        wrapperSimulator.append(simInputOffense);
+
+        const simInputMoneyMin = this.createInput({placeholder: 'Min Money', id: 'minMoney', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputMoneyMin);
+
+        const simInputMoneyMax = this.createInput({placeholder: 'Max Money', id: 'maxMoney', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputMoneyMax);
+
+        const simButton = this.buttonCreate('simulate', 'Simulate');
+        wrapperSimulator.append(simButton);
```
Comment (RaLLy08) [inline src/view.js]: why so much append to WrapperSimulator? please use one append and one const simInputs = this.createInput([
{placeholder: 1 , id: , class: 1, require: true}, {placeholder: 2 , id: 2, class: 2, require: false}] );

**Review by Musa-94 — APPROVED** [review]

**Review by RaLLy08 — APPROVED** [review]
Good work

**Pull request merged**